File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1151.py
fib = [0, 1]
n = int(input())
res = ""
for i in range(2, n):
    fib.append(fib[i-1]+fib[i-2])
for i in range(0, n):
    res=res+str(fib[i])
    if(i==n-1):
        print(res)
    else:
        res=res+" "

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1151.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')

fib = []
fib[0] = 0
fib[1] = 1
n = parseInt(lines.shift())
res = ""
for(i=2;i<n;i++){
    fib[i]=fib[i-1]+fib[i-2]
}
for(i=0;i<n;i++){
    res=res+fib[i]
    if(i==n-1){
        console.log(res)
    } else{
        res=res+" "
    }
}

